### PR TITLE
Add alert for when bz2068601 is effecting a cluster

### PR DESCRIPTION
**Pull request by jewzaam**
upstream: https://github.com/openshift/cluster-etcd-operator/pull/774

goal: get in prod for ROSA/OSD reasonably fast, test/bake in stage first

### CODE REVIEW

File: deploy/bz2068601/OWNERS [hunk 1]
```diff
@@ -0,0 +1,3 @@
+reviewers:
+- jewzaam
+- meta
```
Comment (csheremeta) [inline deploy/bz2068601/OWNERS:3]:
```suggestion
- csheremeta
```

File: deploy/bz2068601/10-etcdDivergentRevisionsSRE.PrometheusRule.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+spec:
+  groups:
+  - name: etcd
+    rules:
+    - alert: etcdDivergentRevisionsSRE
+      annotations:
+        description: etcd is reporting divergent member revisions,
+          with a standard deviation of {{ $value }}.  Members may be
+          stuck or, in extreme circumstances, split brained.
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-etcd-operator/FIXME
```
Comment (wking) [inline deploy/bz2068601/10-etcdDivergentRevisionsSRE.PrometheusRule.yaml:15]: We can optimisitically set `https://github.com/openshift/runbooks/blob/master/alerts/cluster-etcd-operator/etcdDivergentRevisionsSRE.md` here.  Or leave it as FIXME for now; I'm fine either way

Comment (jewzaam) [reply deploy/bz2068601/10-etcdDivergentRevisionsSRE.PrometheusRule.yaml:15]: It won't matter, we generate an SOP link from the alert name.  I'll pair down to just a "message" annotation to be consistent with other alerts we manage.

File: deploy/bz2068601/10-etcdDivergentRevisionsSRE.PrometheusRule.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    - alert: etcdDivergentRevisionsSRE
+      annotations:
+        message: etcd is reporting divergent member revisions,
+          with a standard deviation of {{ $value }}.  Members may be
+          stuck or, in extreme circumstances, split brained.
+      expr: min_over_time(stddev(max by (pod) (etcd_debugging_mvcc_compact_revision{namespace="openshift-etcd"}))[5m:15s]) > 0
+      for: 3m
+      labels:
+        severity: critical
+        namespace: openshift-monitoring
```
Comment (cblecker) [inline deploy/bz2068601/10-etcdDivergentRevisionsSRE.PrometheusRule.yaml:19]:
```suggestion
        namespace: openshift-etcd
```

File: deploy/bz2068601/10-etcdDivergentRevisionsSRE.PrometheusRule.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+spec:
+  groups:
+  - name: etcd
+    rules:
+    - alert: etcdDivergentRevisionsSRE
+      annotations:
+        message: etcd is reporting divergent member revisions,
+          with a standard deviation of {{ $value }}.  Members may be
+          stuck or, in extreme circumstances, split brained.
+      expr: min_over_time(stddev(max by (pod) (etcd_debugging_mvcc_compact_revision{namespace="openshift-etcd"}))[5m:15s]) > 0
```
Comment (cblecker) [inline deploy/bz2068601/10-etcdDivergentRevisionsSRE.PrometheusRule.yaml:15]: Can we split this into a recording rule and then alert if it's `> 0`?

syntax example: https://github.com/openshift/managed-cluster-config/blob/master/deploy/sre-prometheus/100-control-plane-resizing.PrometheusRule.yaml

**Pull request merged**